database/migrations/2022_02_15_204017_create_game_types_table.php

```diff
@@ -17,6 +17,7 @@ public function up()
             $table->id();
             $table->string('title');
             $table->string('description');
+            $table->boolean('mobile_only');
```

Comment:
Žal Laravel deluje tako, da obstoječih migracij nikoli ne poganja. Za to moraš za dodajanje novega stolpca ustvarit novo migracijo.
Torej php artisan make:migration AddMobileOnlyColumnToGameTypesTable in potem notri napišeš samo ta stolpec :)